### PR TITLE
A task to check if only py2 packages contain executables

### PR DESCRIPTION
**Pull request by irushchyshyn**
- [x] check if only py2 packages contain executables
- [x] unit tests
- [x] integration tests

Part of #5

### CODE REVIEW

File: taskotron_python_versions/executables.py [hunk 1]
```diff
@@ -1,8 +1,106 @@
+import collections
+
+from .common import log, write_to_artifact
+
+
+INFO_URL = ''
```
Comment (hroncok) [inline taskotron_python_versions/executables.py]: What about https://fedoraproject.org/wiki/Packaging:Python#Executables_in_.2Fusr.2Fbin

Comment (irushchyshyn) [inline taskotron_python_versions/executables.py]: Right, I forgot to add an actual info link for this. Added

**Comment by hroncok** [comment]
I've pushed a fixup commit, does it look good to you?

File: taskotron_python_versions/executables.py [hunk 2]
```diff
@@ -1,8 +1,109 @@
...
+
+
+INFO_URL = ('https://fedoraproject.org/wiki/Packaging:Python#'
+            'Executables_in_.2Fusr.2Fbin')
+
+MESSAGE = """
+The following executables are available only in Python 2 subpackages:
+{}
+
+If the functionality of those executable is the same regardless of the
```
Comment (irushchyshyn) [inline taskotron_python_versions/executables.py]: typo: those executable**s**?

**Comment by irushchyshyn** [comment]
There is a small typo I think, but your changes look good. Thanks!

**Comment by hroncok** [comment]
Thank You.

**Pull request merged**